react/utils/logger.ts

```diff
@@ -0,0 +1,17 @@
+import axios from 'axios'
+import SplunkEvents from 'splunk-events'
+
+const SPLUNK_ENDPOINT = 'https://splunk-heavyforwarder-public.vtex.com:8088'
+const SPLUNK_TOKEN = '762f39fa-5679-4997-936c-add830df3231'
```

Comment:
Will we allow this splunk token here ? This is an open app, anyone with this token can do anything with our splunk, right ?